app/lib/features/events/pages/event_details_page.dart

```diff
@@ -223,69 +225,60 @@ class _EventDetailPageConsumerState extends ConsumerState<EventDetailPage> {
   }
 
   Widget _buildEventBasicDetails(CalendarEvent calendarEvent) {
-    final month = getMonthFromDate(calendarEvent.utcStart());
-    final day = getDayFromDate(calendarEvent.utcStart());
     final membership = ref
         .watch(roomMembershipProvider(calendarEvent.roomIdStr()))
         .valueOrNull;
     final canPostEvent = membership?.canString('CanPostEvent') == true;
 
-    return Padding(
-      padding: const EdgeInsets.symmetric(horizontal: 20.0),
-      child: Row(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          // Date and Month
-          Column(
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        // Date and Month
+        EventDateWidget(
+          calendarEvent: calendarEvent,
+          size: 80,
```

Comment:
Make `EventDateWidget` as generic widget which will be use in Event Item and Event Detail screen.